ui/src/components/DashboardPlane.test.js

```diff
@@ -39,17 +39,34 @@ jest.mock('../containers/AlertProvider', () => ({
   },
 }));
 
+jest.mock('../containers/ConfigProvider', () => ({
```

Comment:
I don't think the work is related to the commit message :(

And please refer to the best practice of commit messages. 

https://metal-k8s.readthedocs.io/en/latest/developer/development/best_practices/commit.html?highlight=commit#subject